instantly/url.go

```diff
@@ -0,0 +1,18 @@
+package instantly
+
+import "github.com/amp-labs/connectors/common/urlbuilder"
+
+var queryEncodingExceptions = map[string]string{ //nolint:gochecknoglobals
+	// none
+}
+
+func constructURL(base string, path ...string) (*urlbuilder.URL, error) {
```

Comment:
Same as the `WithClient` can we also remove the duplicate definition for `constructURL` and define in `common` package and reuse it?

Reply:
See: https://github.com/amp-labs/connectors/pull/972